reconcile/cli.py

```diff
@@ -109,3 +110,16 @@ def ldap_users(ctx, thread_pool_size):
 def openshift_resources_annotate(ctx, cluster, namespace, kind, name):
     run_integration(reconcile.openshift_resources_annotate.run,
                     ctx.obj['dry_run'], cluster, namespace, kind, name)
+
+
+@integration.command()
+@click.option('--print-only/--no-print-only',
+              default=False,
+              help='If `true`, it will only print the terraform config file.')
+@click.option('--enable-deletion/--no-enable-deletion',
+              default=False,
```

Comment:
as we gain confidence we may want to make the default option to be true in the long term, so can we just do that now and change this to True? we can force the --no-enable-deletion in our pipelines.